tracer/src/Datadog.Trace/Logging/DirectSubmission/DirectLogSubmissionSettings.cs

```diff
@@ -170,8 +170,8 @@ public DirectLogSubmissionSettings(IConfigurationSource? source, IConfigurationT
             ValidationErrors = validationErrors;
             IsEnabled = isEnabled;
 
-            // Logs injection is enabled by default if direct log submission is enabled, otherwise disabled by default
-            LogsInjectionEnabled = config.WithKeys(ConfigurationKeys.LogsInjectionEnabled).AsBool(defaultValue: isEnabled);
+            // Logs injection is enabled by default when not explicitly configured
```

Comment:
nit: we can omit this detail
```suggestion
            // Logs injection is enabled by default
```